packages/edit-post/src/components/header/header-toolbar/index.native.js

```diff
@@ -33,12 +33,41 @@ function HeaderToolbar( {
 	showKeyboardHideButton,
 	getStylesFromColorScheme,
 	onHideKeyboard,
+	isRTL,
 } ) {
 	const scrollViewRef = useRef( null );
 	const scrollToStart = () => {
 		scrollViewRef.current.scrollTo( { x: 0 } );
 	};
 
+	const renderHistoryButtons = () => {
+		const buttons = [
+			/* TODO: replace with EditorHistoryRedo and EditorHistoryUndo */
+			<ToolbarButton
+				key="undoButton"
+				title={ __( 'Undo' ) }
+				icon={ undoIcon }
+				isDisabled={ ! hasUndo }
+				onClick={ undo }
+				extraProps={ {
+					hint: __( 'Double tap to undo last change' ),
+				} }
+			/>,
+			<ToolbarButton
+				key="redoButton"
+				title={ __( 'Redo' ) }
+				icon={ redoIcon }
+				isDisabled={ ! hasRedo }
+				onClick={ redo }
+				extraProps={ {
+					hint: __( 'Double tap to redo last change' ),
+				} }
+			/>,
+		];
+
+		return isRTL ? buttons.reverse() : buttons;
+	};
+
```

Comment:
I have switch `undo/redo` button depending on `isRTL` to rotate the arrows but do not change the direction of the flow on history axis.